### DISCUSSION TITLE
EcoDim.07 not pairing

### DISCUSSION THREAD

**Discussion by rupsen8** [Koenkk/zigbee2mqtt#14937] [Devices]
My stick is working correctly as I just paired two xiaomi sensors.
But when I try to connect te dimmer there is no response in the Z2M log.

I press the reset button on the dimmer for 5 secs, blue light starts flashing, but nothing happends.

The dimmer works with the Tuya app.

I am using a CC2652R1 stick with the coordinator firmware from 2022-02-19 and HomeAssistant

**Comment by tplatenburg** [comment]
I seem to have a similar problem with tube_zb_gw_cc2652p2_v3 as coordinator

Pairing with separate Conbee Phoscon stick went fine. Ordered extra EcoDim.07 to rule out problems with this specific unit. I do have other successful pairs with EcoDim.07.

**Comment by rupsen8** [comment]
I went and got another dimmer, this one works so it was probably a faulty dimmer.

**Reply by tplatenburg** [reply]
Thanks for your reply, something more to sort out on my side then!
